crates/evm/src/query.rs

```diff
@@ -90,7 +90,7 @@ pub struct EstimatedDiffSize {
 #[rpc_gen(client, server)]
 impl<C: sov_modules_api::Context> Evm<C> {
     /// Handler for `net_version`
-    #[rpc_method(name = "net_version")]
+    #[rpc_method(name = "net_version", blocking)]
```

Comment:
This marker basically tells jsonrpsee that we want to `spawn_blocking` this call and await it internally.